src/AWS/Lambda/Lambda.purs

```diff
@@ -17,31 +22,18 @@ type InternalLambdaParams = {
 
 foreign import data Lambda :: Type
 
-type InternalMakeClientParams
-  = { region :: String
-    , secretAccessKey :: String
-    , accessKeyId :: String
-    }
+foreign import newLambda :: Foreign -> (Effect Lambda)
 
-foreign import makeClientImpl :: InternalMakeClientParams -> (Effect Lambda)
+makeClient :: forall t4 t5 t6 t7 t8.
+  RowToList t6 t5 => FillableFields t5 () t6 => Union t8 t6
+                                                  DefaultClientPropsR
+                                                 => RowToList t7 t4 => JustifiableFields t4 t7 () t8 => Record t7 -> Effect Lambda
+makeClient r = ((makeDefaultClient r:: DefaultClientProps)) # makeClientHelper newLambda
 
-foreign import makeDefaultClientImpl :: Effect Lambda
+foreign import invokeImpl :: forall output. Fn2 Lambda InternalLambdaParams (Effect (Promise output))
 
-makeClient :: Region -> AccessKeyId -> SecretAccessKey -> Effect Lambda
-makeClient r a s =
-  makeClientImpl
-    { region: un Region r
-    , secretAccessKey: un SecretAccessKey s
-    , accessKeyId: un AccessKeyId a
-    }
-
-makeDefaultClient :: Effect Lambda
-makeDefaultClient = makeDefaultClientImpl
-
-foreign import invokeFunctionImpl :: forall output. Fn2 Lambda InternalLambdaParams (Effect (Promise output))
-
-invokeFunction :: forall input output. WriteForeign input => Lambda -> Arn -> input -> Aff output
-invokeFunction client (Arn arn) input = runFn2 invokeFunctionImpl client params # toAffE
+invoke :: forall input output. WriteForeign input => Lambda -> Arn -> input -> Aff output
```

Comment:
Reminder: update the call in the CostExplorer repo (Main.purs of the Importer-proxy)